js/screenSnippet/index.js

```diff
@@ -61,6 +63,17 @@ class ScreenSnippet {
                     captureUtil = path.join(execPath, 'ScreenSnippet.exe');
                 }
 
+                // Method to verify and disable always on top property
+                // as an issue with the ScreenSnippet.exe not being on top
+                // of the electron wrapper
+                const windows = electron.BrowserWindow.getAllWindows();
+                if (windows && windows.length > 0) {
+                    isAlwaysOnTop = windows[ 0 ].isAlwaysOnTop();
```

Comment:
Is it fine if i read this from electron api or should i read it from config file?

@VikasShashidhar, @VishwasShashidhar & @lneir

Reply:
Either way should be fine. I'd prefer using the electron api as it is dynamic.